utils/constants/common-constants.ts

```diff
@@ -0,0 +1,33 @@
+import { HomeIcon, BookIcon, SettingIcon } from '../../assets/icons';
+
+interface SidebarItem {
```

Comment:
We have a folder `models` to keep data types or interfaces. I believe it is better to move this interface to there and import here.